README.md

```diff
@@ -1,6 +1,24 @@
 # Welcome to Crowbar Angular Prototype
 [![Build Status](https://travis-ci.org/crowbar/crowbar-ui.svg?branch=master)](https://travis-ci.org/crowbar/crowbar-ui)
 
+### Setting up a development environment with Docker
+
+This is an optional way to develop on `crowbar-ui` within a contained environment. The main benefit is to have some reusable and host-independent environment to develop on.
+
+1. make sure `docker` is installed and the `docker` daemon is running.
+
+   for installation details see: https://docs.docker.com/engine/installation/
+2. build the image from the [Dockerfile](Dockerfile):
+  ```
+  docker build -t crowbar-ui .
+  ```
+3. run the container (`pwd` should be your git clone):
+  ```
+  docker run -d --name "crowbar-ui_container" -v $(pwd)/assets:/crowbar-ui/assets -v $(pwd)/routes:/crowbar-ui/routes -p 3000:3000 -p 3001:9876 crowbar-ui
```

Comment:
oops, yes, that still is a remainder of trying to get the coverage static files served... and karma seems to have some open port in the config, and i thought it is a webserver for the coverage, but it didn't work somehow...